### PR TITLE
Adds lines 162-192 for open source contribution

### PR DESCRIPTION
**Pull request by lamagnifica**
going in today’s release
https://github.com/edx/edx-platform/pull/8167

### CODE REVIEW

**Comment by sarina** [comment]
@lamagnifica One request, please don't @ mention github names in commit messages. I will now get an email anytime anyone rebases their branch and recommits this commit (even on forked repos!!) I'm still getting emails from a commit I was @ mentioned in on edx-platform 8 months ago.

**Comment by lamagnifica** [comment]
ah, I had no idea. Apologies, Sarina

**Comment by lamagnifica** [comment]
@mhoeber @sarina please review

File: en_us/shared/building_and_running_chapters/running_course/course_enrollment.rst [hunk 1]
```diff
@@ -156,37 +159,70 @@ To view the enrollment count for a course:
 You can also view or download a list of the people who are enrolled in the
 course. See :ref:`Student Data`.
 
+.. _view_not_yet_enrolled:
+
+********************************
+Report Learners Not Yet Enrolled
+********************************
+
+After you enroll learners in a course using the **Auto Enroll** option, those
```
Comment (sarina) [inline en_us/shared/building_and_running_chapters/running_course/course_enrollment.rst]: I wonder if it's worth rephrasing to emphasize this is for learners who don't have accounts. Eg, "When enrolling learners in a course who don't yet have an edX account, those learners must complete the account registration and activation process to finish the enrollment process." (if you disagree I obviously defer to your wisdom... I'm learning from my trivia mistakes! :smile:  )

File: en_us/shared/building_and_running_chapters/running_course/course_enrollment.rst [hunk 2]
```diff
@@ -156,37 +159,70 @@ To view the enrollment count for a course:
...
+learners must complete the account registration and activation process to
+complete the enrollment process. You can download a report of auto enrolled
```
Comment (sarina) [inline en_us/shared/building_and_running_chapters/running_course/course_enrollment.rst]: You use "complete" & "process" 2x in this sentence ("_complete_ the account registration and activation **process** to _complete_ the enrollment **process**")

**Comment by sarina** [comment]
@lamagnifica no worries I honestly didn't know until that happened to me. Just something to keep in mind going forward.

LGTM :+1:

File: en_us/shared/building_and_running_chapters/running_course/course_enrollment.rst (hunk 2 above)
Comment (mhoeber) [inline en_us/shared/building_and_running_chapters/running_course/course_enrollment.rst]: ". . those learners must register and activate their accounts to complete the enrollment process."  ?

**Comment by mhoeber** [comment]
:+1:

**Pull request merged**